test/test.py

```diff
@@ -450,6 +459,8 @@ def _assert_bundle(self, uuid, version, manifest, metadata_files,
         self.assertEqual(geo_series_accessions, project.geo_series_accessions)
         self.assertEqual(array_express_accessions, project.array_express_accessions)
         self.assertEqual(insdc_study_accessions, project.insdc_study_accessions)
+        if self.expected_accessions:
```

Comment:
@hannes-ucsc 
I opted to add the instance variable in the setup, an alternative would have been  
`if getattr(self, 'expected_accessions`, None)` but seemed a bit awkward.